CRM/Utils/System/Drupal8.php

```diff
@@ -56,6 +56,9 @@ public function createUser(&$params, $mail) {
     // Validate the user object
     $violations = $account->validate();
     if (count($violations)) {
+      foreach ($violations as $violation) {
```

Comment:
@demeritcowboy do we need to do this on other CMSes?

Reply:
For drupal 7 no, because of the way it uses drupal's form layer to create a user, so the form errors display.

Let's take a look at what wordpress/joomla do...

Reply:
Ok it took a minute to find but it looks like wordpress doesn't override checkPermissionAddUser() so it returns false so you don't even see this feature.

Ok now joomla...

Reply:
The Joomla UF class has pretty extensive checking as a form rule via checkUserNameEmailExists(), and it's beyond my joomla knowledge to know how to get past it to trigger a fail in joomla itself. The only thing I can see it doesn't check for is a blank email, which this PR would cover. But createUser itself doesn't have any kind of error checking, and in fact seems like it would never return FALSE either way, so technically createUser might need some sprucing up, but I'm not a joomla person.